build-system/tasks/runtime-test/helpers.js

```diff
@@ -192,10 +192,16 @@ function unitTestsToRun(unitTestPaths) {
   }
 
   filesChanged.forEach(file => {
+    if (!fs.existsSync(file)) {
+      if (!process.env.TRAVIS) {
+        log(green('INFO:'), 'Skipping', cyan(file), 'because it was deleted');
+      }
+      return;
```

Comment:
nit (feel free to ignore if you just want to get this in) - since this is one of those long "if/else" functions, you can remove the "return" statement and change the next conditional to be an "else if" instead

Reply:
Personal preference is to always favor guard clauses but I'll keep consistency here ;)